.env.example

```diff
@@ -18,6 +15,15 @@ LOCAL_POSTGRES_PASSWORD=postgrespassword
 LOCAL_POSTGRES_DATABASE=postgres
 LOCAL_HASURA_ADMIN_SECRET=admin-secret
 
+# Hardhat
+HARDHAT_CHAIN_ID=1337
+HARDHAT_PORT=8545
+HARDHAT_GANACHE_CHAIN_ID=1338
+HARDHAT_GANACHE_PORT=8546
+HARDHAT_FORK_BLOCK=13500000
+HARDHAT_OWNER_ADDRESS=0xYOUR_DEV_ADDRESS
```

Comment:
I'm so lazy, I wish this would use the LOCAL_SEED_ADDRESS here too by default

Reply:
yeah i should probably combine those